components/molecules/HighlightInput/highlight-input-form.tsx

```diff
@@ -153,7 +153,7 @@ const HighlightInputForm = (): JSX.Element => {
       {pullrequestLink && isDivFocused && <GhOpenGraphImg githubLink={pullrequestLink} />}
 
       {isDivFocused && (
-        <Button loading={loading} disabled={!bodyText || !validCharLimit()} className="ml-auto" variant="primary">
+        <Button loading={loading} disabled={!bodyText || !validCharLimit()} className="ml-auto " variant="primary">
```

Comment:
stray spacing and some other eslint issues could be solved by applying some of the auto-format rules from the backends (we are ultimately using typescript the same way outside of react), don't remember if there was an issue for this 😅